packages/shorebird_cli/test/src/auth/auth_test.dart

```diff
@@ -1,16 +1,18 @@
 import 'dart:convert';
-import 'dart:io';
+import 'dart:io' hide Platform;
```

Comment:
I wonder if we need dart:io at all or if there is some other mocking layer we're missing?